pulsar-io/hbase/pom.xml

```diff
@@ -70,12 +70,6 @@
             <version>${hbase.version}</version>
         </dependency>
 
-        <dependency>
```

Comment:
this is patch of another patch

Reply:
yes I cherry-picked it since the fix for the broken master branch wasn't merged yet when I pushed this PR. I'll rebase (the commit would get remove) this PR.